.github/workflows/ci.yml

```diff
@@ -16,6 +16,30 @@ jobs:
         components: rustfmt
     - name: Check Formatting
       run: cargo +stable fmt --all -- --check
+  
+  cargo-deny:
+    name: cargo-deny
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - aarch64-apple-darwin
+          - aarch64-apple-darwin
+          - aarch64-linux-android
+          - x86_64-apple-darwin
+          - x86_64-pc-windows-msvc
+          - x86_64-unknown-linux-gnu
+          - x86_64-unknown-linux-musl
```

Comment:
In general, could we match the targets were actually testing? You're at least missing the `wasm32-unknown-unknown`, not sure whether we need a special case for musl.